content/_policy/rules-of-use._es.md

```diff
@@ -7,158 +7,157 @@ permalink: /es/policy/rules-of-use/
 
-Puede cancelar su cuenta en cualquier momento a través de su perfil de cuenta. Además, en caso de fraude u otras violaciones de estas normas de uso, podemos revocar el acceso a su cuenta.
+## 5. Rescisión
 
-Si esto ocurre, seguiremos protegiendo la información de su cuenta de acuerdo con nuestra [política de privacidad]({% link _policy/index._es.md %}) y nuestro aviso de sistema de registro [aquí](https://www.federalregister.gov/documents/2017/01/19/2017-01174/privacy-act-of-1974-notice-of-a-new-system-of-records) y [aquí](https://www.federalregister.gov/documents/2017/08/10/2017-16852/privacy-act-of-1974-system-of-records).
+Puede cancelar su cuenta en cualquier momento a través de su perfil de cuenta. Además, en caso de fraude u otras vulneraciones de estas Normas de uso, podemos revocar el acceso a su cuenta. Si esto ocurre, seguiremos protegiendo la información de su cuenta de acuerdo con nuestra [Política de privacidad]({% link _policy/index._es.md %}) y el Aviso sobre el Sistema de Registro [aquí](https://www.federalregister.gov/documents/2022/11/21/2022-25420/privacy-act-of-1974-notice-of-a-modified-system-of-records).
 
-En el improbable caso de que se interrumpa el servicio de Login.gov, la información de la cuenta se conservará con arreglo a nuestro aviso sobre el sistema de registros, en su versión modificada, al que puede acceder [aquí](https://www.federalregister.gov/documents/2017/01/19/2017-01174/privacy-act-of-1974-notice-of-a-new-system-of-records)y [aquí](https://www.federalregister.gov/documents/2017/08/10/2017-16852/privacy-act-of-1974-system-of-records). Concretamente, la información de los usuarios, incluidos los perfiles, los archivos de inicio de sesión, los archivos de contraseñas, los archivos y extractos de registros de auditoría, los archivos de uso del sistema y los archivos de costos utilizados para evaluar los cargos por el uso del sistema, se destruyen 6 años después de la alteración de la contraseña o de la cancelación de la cuenta de usuario, a menos que se requiera una retención más prolongada para uso empresarial, como por ejemplo para asuntos de aplicación de la ley u otras acciones legales.
+En el improbable caso que se interrumpa el servicio Login.gov, la información de la cuenta se conservará sujeta a nuestra Notificación sobre el sistema de registros, en su versión modificada, a la que puede acceder [aquí]((https://www.federalregister.gov/documents/2022/11/21/2022-25420/privacy-act-of-1974-notice-of-a-modified-system-of-records). De forma específica, la información de los usuarios, incluidos los perfiles, los archivos de acceso, los archivos de contraseñas, los archivos y extractos de registros de auditoría, los archivos de uso del sistema y los archivos de devolución de costos utilizados para evaluar los cargos por el uso del sistema se conservan de acuerdo con el Programa de Retención de Registros de la GSA, GRS 03.2.
```

Comment:
Small formatting error here causing the text not be linked:

![image](https://user-images.githubusercontent.com/1779930/225712065-7609e474-9e67-49a7-a51a-94c464c62a2e.png)

```suggestion
En el improbable caso que se interrumpa el servicio Login.gov, la información de la cuenta se conservará sujeta a nuestra Notificación sobre el sistema de registros, en su versión modificada, a la que puede acceder [aquí](https://www.federalregister.gov/documents/2022/11/21/2022-25420/privacy-act-of-1974-notice-of-a-modified-system-of-records). De forma específica, la información de los usuarios, incluidos los perfiles, los archivos de acceso, los archivos de contraseñas, los archivos y extractos de registros de auditoría, los archivos de uso del sistema y los archivos de devolución de costos utilizados para evaluar los cargos por el uso del sistema se conservan de acuerdo con el Programa de Retención de Registros de la GSA, GRS 03.2.
```